content/en/docs/ops/configuration/traffic-management/dns-proxy/index.md

```diff
@@ -0,0 +1,113 @@
+---
+title: DNS Proxying
+description: How to configure DNS proxying.
+weight: 60
+keywords: [traffic-management,dns,virtual-machine]
+owner: istio/wg-networking-maintainers
+test: no
+---
+
+{{< warning >}}
+This feature is actively in [development](https://github.com/istio/community/blob/master/FEATURE-LIFECYCLE.md) and is considered `alpha`.
```

Comment:
According to the referenced feature stages page, `development` and `alpha` contradict each other. Is this feature alpha or pre-alpha (development).
```suggestion
This feature is actively in [development](https://github.com/istio/community/blob/master/FEATURE-LIFECYCLE.md) and is considered `pre-alpha`.
```

Reply:
Or:
```suggestion
This feature is currently [experimental](https://github.com/istio/community/blob/master/FEATURE-LIFECYCLE.md) and considered `alpha`.
```